utils/Common.py

```diff
@@ -55,3 +56,10 @@ def exec_command(context, command, shell=False, stdout_msg=None):
             raise Exception(stderr)
 
 
+def set_metadata(context, file, fdict):
+    # In the event that there is a full directory in front of the file name, remove it.
```

Comment:
Removed by `.name`? Just checking that the directory doesn't need to get added back in l64, when file2 is sent to the dictionary...

Reply:
yes, it needed to be stripped in both places to work

Reply:
which is done on line 44...this is a very old gear I would not lay it out this way anymore.  It deserves more of a facelift but maybe another sprint.